src/ui/libs/DatalensChartkit/ChartKit/plugins/Table/renderer/components/Table/Table.tsx

```diff
@@ -237,7 +228,7 @@ export const Table = React.memo<Props>((props: Props) => {
                     )}
                     {!noData && (
                         <table
-                            className={b({prepared: !prerender})}
+                            className={b({prepared: true})}
```

Comment:
Do we actually need this modifier?

Reply:
We definitely need some kind of modifier - tables need different styles (one background for minimum column sizes and one for displaying to the user). But maybe it should be called differently, since the code has undergone changes